doc/BUG-REPORT

```diff
@@ -42,17 +42,9 @@ DO NOT SEND HTML E-MAIL TO THE LISTS.
 2) INFORMATION ABOUT TCL
 
 2.1) Tcl library version:
-     ( ) 8.3
-     ( ) 8.4
-     ( ) 8.5
-     ( ) 8.6
-     ( ) Other - Which? ____
+     8.6._
```

Comment:
We still support those versions, at least 8.5 and 8.4 for sure

Reply:
alright